utilities/console.py

```diff
@@ -108,10 +115,44 @@ def __enter__(self):
         """
         Connect to console
         """
-        return self.connect()
+        self.connect()
+        return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         """
         Logout from shell
         """
         self.disconnect()
+
+    def run_commands(
+        self,
+        commands: list[str],
+        timeout: int = TIMEOUT_1MIN,
+        verify_commands_output: bool = True,
+        command_output: bool = False,
```

Comment:
I don't see the benefit of `command_output: bool = False` 
I think you can always return the output.

Reply:
I agree, but I was unsure about the original need of it.
I am in favor to remove it now if @dbasunag also agrees.

Reply:
It may require you to change the calls to this function. As long as you are fine with handling that, I have no issues with dropping the argument. Also if we always return output, then I don't see the need of `verify_commands_output`.

Reply:
`verify_commands_output` raises in case of retval != 0.
I do not think we want to ignore it, but it seems some executions do want to ignore it.
I prefer this is kept now and considered for removal later.

The `command_output` is easy to remove because no one uses it now.